### DISCUSSION TITLE
No BING photogrammetry

### DISCUSSION THREAD

**Discussion by 3DSpotters** [derekhe/msfs2020-google-map#39] [General]
So I've checked:

Google mod ON - no BING photogrammetry :/ ...
https://ibb.co/nRkNhpJ

Google mod OFF - BING photogrammetry is ON!
https://ibb.co/pdhMhY8

Looks like the above behavior is logical but... is it possible to somehow do workaround this problem, from inside this mod?

**Comment by derekhe** [comment]
Please try latest version, fixed.

**Comment by 3DSpotters** [comment]
CHECKED, DONE, IT'S WORKING, WONDERFUL!

https://ibb.co/gzjxbMv
